letsencrypt-win-simple/Program.cs

```diff
@@ -664,9 +671,23 @@ public static void EnsureTaskScheduler()
                     var currentExec = Assembly.GetExecutingAssembly().Location;
 
                     // Create an action that will launch the app with the renew paramaters whenever the trigger fires
-                    task.Actions.Add(new ExecAction(currentExec, $"--renew --baseuri \"{BaseURI}\"", Path.GetDirectoryName(currentExec)));
+                    var arguments = $"--renew --baseuri \"{BaseURI}\"";
+                    if (Options.User)
+                    {
+                        arguments += " --user";
+                    }
+                    if (Options.Test)
```

Comment:
You don't need to add the --test option as the renewal is done via the used BaseUri. If the BaseUri changes for the --test option, doing it this way it won't be a renewal but it will generate a new certificate signed by a different source.